includes/admin/views/omise-page-settings.php

```diff
@@ -98,85 +98,91 @@
 		<hr />
 
 		<h3><?php _e( 'Payment Methods', 'omise' ); ?></h3>
-		<p><?php _e( 'The table below is a list of available payment methods that you can enable in your WooCommerce store.', 'omise' ); ?></p>
-		<table class="form-table">
-			<tbody>
-				<tr>
-					<th scope="row"><label for="sandbox"><?php _e( 'Available Payment Methods', 'omise' ); ?></label></th>
-					<td>
-						<table class="widefat fixed striped" cellspacing="0">
-							<thead>
-								<tr>
-									<?php
-										$columns = array(
-											'name'    => __( 'Payment Method', 'omise' ),
-											'status'  => __( 'Enabled', 'omise' ),
-											'setting' => ''
-										);
+		<?php if ($settings['account_country']) : ?>
```

Comment:
Note; this seems to be a lot of changes but it only is because of this line and another following line.
The rest is about the changing of its indent.